1. Contributing/Reporting Issues/README.md

```diff
@@ -1,57 +1,57 @@
-# Reporting Issues
+# Bug Reporting
 
-You don't have to write code to contribute! We are always looking for help with creating or updating documentation, testing our apps (web, desktop and mobile) and reporting bugs.
+## Is it a bug?
 
-## Be sure its a bug
-We are striving to make our issues a list of actionable items for the team.  So we want to keep it a list of issues and feature requests.
+GitHub Issues are for bugs and feature requests only.
 
 Examples of things that aren't considered bugs or feature requests:
+
 * Can't figure out how to do something
 * Having issues accessing your server
 * Can't get the server installed
 
-If your issue isn't a bug please see [Getting Support](/2. Getting%20Support)
+If your issue falls into one of these categories, please see [Getting Support](/2. Getting%20Support)
 
-## Reporting a bug
+## It is a bug!
 
-Did you find a bug? Reporting is easy!
+### Before reporting a new issue
 
 1. [Search the Issues](https://github.com/RocketChat/Rocket.Chat/issues) for your bug.  If you find it, give it a thumbsup and add some additional information to the ticket if differing.  This will help give us a fuller picture.
```

Comment:
I have put back this section since it's important to our issue workflow on the main repo.